docs/develop/add-ins-with-angular2.md

```diff
@@ -71,27 +71,24 @@ const routes: Routes = // route definitions go here
 export class AppRoutingModule { }
 ```
 
-
 ## Consider wrapping Fabric components with Angular components
 
-We recommend using [UI Fabric](https://developer.microsoft.com/fabric#) styling in your add-in. UI Fabric for the web is available in two flavors: 
+We recommend using [UI Fabric](https://developer.microsoft.com/fabric#) styling in your add-in. UI Fabric for the web is available in two flavors:
```

Comment:
Fabric is deprecated. This link redirects to Fluent UI. This section should be rewritten because Fabric is now out of support.

Reply:
Actually, let's remove all references to Fabric or Fluent from this article. Fluent UI is React-based and we don't want to encourage people to mix React with Angular.